drop-down.ios.ts

```diff
@@ -353,17 +371,49 @@ class DropDownListPickerDelegateImpl extends NSObject implements UIPickerViewDel
         if (style.color) {
             label.textColor = style.color.ios;
         }
+        
+        let itemsPaddingTop = owner.effectivePaddingTop;
+        let itemsPaddingRight = owner.effectivePaddingRight;
+        let itemsPaddingBottom = owner.effectivePaddingBottom;
+        let itemsPaddingLeft = owner.effectivePaddingLeft;
+        if (owner.nativeView.itemsPadding !== itemsPaddingProperty.defaultValue) { 
+            const itemsPadding = owner.nativeView.itemsPadding.split(" ").map((s) => parseInt(s, 10));
+            if (itemsPadding.length === 1) {
+                itemsPaddingTop = itemsPadding[0];
+                itemsPaddingRight = itemsPadding[0];
+                itemsPaddingBottom = itemsPadding[0];
+                itemsPaddingLeft = itemsPadding[0];
+            } else if (itemsPadding.length === 2) {
+                itemsPaddingTop = itemsPadding[0];
+                itemsPaddingRight = itemsPadding[1];
+                itemsPaddingBottom = itemsPadding[0];
+                itemsPaddingLeft = itemsPadding[1];
+            } else if (itemsPadding.length === 3) {
+                itemsPaddingTop = itemsPadding[0];
+                itemsPaddingRight = itemsPadding[1];
+                itemsPaddingBottom = itemsPadding[2];
+                itemsPaddingLeft = itemsPadding[1];
+            } else if (itemsPadding.length === 4) {
+                itemsPaddingTop = itemsPadding[0];
+                itemsPaddingRight = itemsPadding[1];
+                itemsPaddingBottom = itemsPadding[2];
+                itemsPaddingLeft = itemsPadding[3];
+            }
+        }
```

Comment:
This whole logic will have to be refactored to accommodate couple of things:
1. In NS it is supported to enter padding separated by `,`. So a similar function to https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/styling/style-properties.ts#L379 should be used
2. Paddings are not only numbers, they can also be `px`, etc. So you cant use `parseInt`. You must use a similar approach to https://github.com/NativeScript/NativeScript/blob/4f39fb728b6bcacdafed38dd0a80137fcc9fb1e9/tns-core-modules/ui/styling/style-properties.ts#L448 and use `Length.parse` to get the actual value.